### PR TITLE
Updated release notes AzureML-container

### PR DESCRIPTION
**Pull request by v-tashish**

### CODE REVIEW

File: base/gpu/openmpi4.1.0-cuda11.3-cudnn8-ubuntu20.04/release-notes.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+-   **Dependencies** :
+
+-   [CVE-2019-20838](https://ubuntu.com/security/notices/USN-5425-1) 
+-   [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5446-1)
+-   [CVE-2022-1304](https://ubuntu.com/security/notices/USN-5464-1) 
+-   [CVE-2022-2068](https://ubuntu.com/security/notices/USN-5488-1)
+-   [CVE-2022-32208](https://ubuntu.com/security/notices/USN-5495-1) 
+
+- released images to mitigate PCRE, dpkg, CURL,OpenSSL and E2fsprogs Vulnerabilities.
+
```
Comment (Kmumuda) [inline base/gpu/openmpi4.1.0-cuda11.3-cudnn8-ubuntu20.04/release-notes.md:15]: remove these extra lines here

File: base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu18.04/release-notes.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
+-  [CVE-2019-20838](https://ubuntu.com/security/notices/USN-5425-1)
+-  [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5446-1)
 -  [CVE-2019-13050](https://ubuntu.com/security/notices/USN-5431-1)
--  [CVE-2020-14155](https://ubuntu.com/security/notices/USN-5425-1)
--  [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5464-1)
 -  [CVE-2022-1304](https://ubuntu.com/security/notices/USN-5464-1)
+-  [CVE-2022-2068](https://ubuntu.com/security/notices/USN-5488-1)
+-  [CVE-2022-32208](https://ubuntu.com/security/notices/USN-5495-1)
 
-- released images to mitigate GnuPG,PCRE,dpkg and E2fsprogs Vulnerabilities.
```
Comment (Kmumuda) [inline base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu18.04/release-notes.md:12]: dont overwrite the old values add new values

File: base/gpu/openmpi4.1.0-cuda11.2-cudnn8-ubuntu20.04/release-notes.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+
+-   [CVE-2019-20838](https://ubuntu.com/security/notices/USN-5425-1) 
+-   [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5446-1)
+-   [CVE-2022-1304](https://ubuntu.com/security/notices/USN-5464-1) 
+-   [CVE-2022-32207](https://ubuntu.com/security/notices/USN-5495-1)
+-   [CVE-2022-2068](https://ubuntu.com/security/notices/USN-5488-1)
+
+     - released images to mitigate PCRE, dpkg, OpenSSL, CURL and E2fsprogs Vulnerabilities.
+
+
```
Comment (Kmumuda) [inline base/gpu/openmpi4.1.0-cuda11.2-cudnn8-ubuntu20.04/release-notes.md:16]: check the alignment for this ...-releases

File: base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu20.04/release-notes.md [hunk 4]
```diff
@@ -1,14 +1,16 @@
...
--   [CVE-2020-14155](https://ubuntu.com/security/notices/USN-5425-1) 
--   [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5464-1)
+-   [CVE-2019-20838](https://ubuntu.com/security/notices/USN-5425-1) 
+-   [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5446CVE-2022-1304-1)
 -   [CVE-2022-1304](https://ubuntu.com/security/notices/USN-5464-1) 
+-   [CVE-2022-2068](https://ubuntu.com/security/notices/USN-5488-1) 
+-   [CVE-2022-32208](https://ubuntu.com/security/notices/USN-5495-1)
 
-- released images to mitigate PCRE, dpkg and E2fsprogs Vulnerabilities.
+- released images to mitigate PCRE, dpkg, OpenSSL, CURL and E2fsprogs Vulnerabilities.
```
Comment (Kmumuda) [inline base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu20.04/release-notes.md:13]: dont overwrite the old values add new values

File: base/gpu/openmpi4.1.0-cuda11.2-cudnn8-ubuntu20.04/release-notes.md [hunk 5]
```diff
@@ -1,5 +1,19 @@
...
+-   **Dependencies** :
+------------------- 
+
+-   [CVE-2019-20838](https://ubuntu.com/security/notices/USN-5425-1) 
+-   [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5446-1)
+-   [CVE-2022-1304](https://ubuntu.com/security/notices/USN-5464-1) 
+-   [CVE-2022-32207](https://ubuntu.com/security/notices/USN-5495-1)
+-   [CVE-2022-2068](https://ubuntu.com/security/notices/USN-5488-1)
+
+     - released images to mitigate PCRE, dpkg, OpenSSL, CURL and E2fsprogs Vulnerabilities.
```
Comment (Kmumuda) [inline base/gpu/openmpi4.1.0-cuda11.2-cudnn8-ubuntu20.04/release-notes.md:14]: check this alignment

File: base/cpu/openmpi4.1.0-spark3-ubuntu20.04/Dockerfile [hunk 6]
```diff
@@ -44,7 +44,8 @@ RUN mkdir -p /usr/share/man/man1 && \
     fuse && \
     apt-get clean -y && \
     rm -rf /var/lib/apt/lists/*
-
```
Comment (Kmumuda) [inline base/cpu/openmpi4.1.0-spark3-ubuntu20.04/Dockerfile:47]: remove this ...image from PR

File: base/cpu/openmpi4.1.0-ubuntu20.04/release-notes.md [hunk 7]
```diff
@@ -10,7 +23,7 @@
 -   [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5464-1) 
 -   [CVE-2022-1304](https://ubuntu.com/security/notices/USN-5464-1)  
 
-- released images to mitigate libxml2,PCRE, OpenLDAP dpkg and E2fsprogs Vulnerabilities.
+- released images to mitigate libxml2, PCRE, OpenLDAP dpkg and E2fsprogs Vulnerabilities.
```
Comment (Kmumuda) [inline base/cpu/openmpi4.1.0-ubuntu20.04/release-notes.md:26]: Check this why you have added all these here ?dont overwrite the old values add new values

**Comment by v-tashish** [comment]
Closing this PR

**Pull request closed without merging**